themes/pom.xml

```diff
@@ -131,6 +86,72 @@
     </build>
 
     <profiles>
+        <profile>
+            <id>account2</id>
+            <activation>
+                <!-- 'activeByDefault' will not work here as other profiles in this POM are always active, so we need to have a workaround to activate this profile. -->
+                <file>
```

Comment:
Use properties to activate this like this:
https://github.com/keycloak/keycloak/blob/bbe9ab38bc6afeaac95c995efd0e081aeb9b0e06/pom.xml#L2038

Reply:
This approach is consistent with what we do elsewhere and also useful for Devs in this repo to disable account console build if they want to save some time.